moveit_ros/planning_interface/CMakeLists.txt

```diff
@@ -23,7 +24,15 @@ find_package(tf2_geometry_msgs REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rclpy REQUIRED)
-find_package(Boost REQUIRED)
+find_package(Boost REQUIRED COMPONENTS
```

Comment:
We have to leave it this way

Reply:
I doubt we need to use the `COMPONENTS` keyword. Can you justify why?

Reply:
We just have to use ``find_package(Boost REQUIRED)`` we cannot split the components into ``find_package(date_time REQUIRED)``